roles/common/tasks/ipmi.yml

```diff
@@ -6,7 +6,7 @@
   apt: pkg=ipmitool
 
 - name: load kernel ipmi modules
-  modprobe: name={{ item }} state=probe
+  modprobe: name={{ item }} state=absent
```

Comment:
We have a custom modprobe library that adds the "probe" state. If your testing is not finding this, then it isn't discovering the modules in ursula's library/ subdirectory.